class/defaults.yml

```diff
@@ -5,6 +5,7 @@ parameters:
     # See: https://github.com/projectsyn/commodore/issues/71
     enabled: true
     namespace: syn-cluster-backup
+    etcd_backup_namespace: ${cluster_backup:namespace}-etcd
```

Comment:
There's an argument to not even expose this parameter to users, and instead always use `'%s-etcd' % params.namespace` for the etcd backup namespace.

Reply:
I think this is the way to go. There's no need to configure the name IMO.